padertorch/data/fragmenter.py

```diff
@@ -122,7 +122,7 @@ def fragment(key, x):
 
 
 def get_anchor(
-        num_samples: int, length: int, shift: int=None,
+        num_samples: int, length: int, shift: int = None,
         mode: str = 'left', rng=np.random
```

Comment:
How about supporting the new numpy 1.19 random number generator `np.random.default_rng()`? (https://numpy.org/doc/stable/reference/random/new-or-different.html) We would need a typecheck for rng and use `integers` instead of `randint`.

Reply:
Can you explain the advantage of the new random number generator?
They did some API cleanup and changed the internals, but I don't see the short term advantage.
The short term disadvantage is, that we would require at least numpy 1.19.

Reply:
I think the idea is to support both the current and the future version of random by doing a typecheck on rng, so we would only need 1.19 if the old numpy typecheck did not work.

Reply:
As far as I can see, the main advantage is that they use a better random number generator algorithm.

Reply:
And at some point, they are probably going to drop support for the old `RandomState`.